commit 5092e57cd950d62b4519f0c12a481e66f18621ab
Author: Otto Pflüger <otto.pflueger@abscue.de>
Date:   Mon Jun 26 14:50:21 2023 +0200

    HACK: wcn36xx: disable 802.11n WiFi

diff --git a/drivers/net/wireless/ath/wcn36xx/main.c b/drivers/net/wireless/ath/wcn36xx/main.c
index 8dbd115a393c..2d4de9427a53 100644
--- a/drivers/net/wireless/ath/wcn36xx/main.c
+++ b/drivers/net/wireless/ath/wcn36xx/main.c
@@ -141,7 +141,7 @@ static struct ieee80211_supported_band wcn_band_2ghz = {
 			IEEE80211_HT_CAP_LSIG_TXOP_PROT |
 			IEEE80211_HT_CAP_SGI_40 |
 			IEEE80211_HT_CAP_SUP_WIDTH_20_40,
-		.ht_supported = true,
+		.ht_supported = false,
 		.ampdu_factor = IEEE80211_HT_MAX_AMPDU_64K,
 		.ampdu_density = IEEE80211_HT_MPDU_DENSITY_16,
 		.mcs = {
